apps/s_server.c

```diff
@@ -147,13 +147,14 @@ static int dtlslisten = 0;
 static char *psk_identity = "Client_identity";
 char *psk_key = NULL;           /* by default PSK is not used */
 
-static unsigned int psk_server_cb(SSL *ssl, const char *identity,
+static unsigned int psk_server_cb(SSL *s, const char *identity,
```

Comment:
Just to clarify. It should be obvious from what was said yesterday, but "argument name matching is not in contradiction with style guide in cases of indirection like above" came out wrong, as I meant "argument *renaming* is not in contradiction [in cases of *indirection*]".